From 035ae6fd6bd1046c4c16f85404b7a4f0d7ee1d0c Mon Sep 17 00:00:00 2001
From: Jan Janssen <janssen@mpie.de>
Date: Sat, 21 Sep 2024 07:16:47 +0200
Subject: [PATCH] add requirements

---
 requirements.txt | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 requirements.txt

diff --git a/requirements.txt b/requirements.txt
new file mode 100644
index 0000000..f05a5c0
--- /dev/null
+++ requirements.txt
@@ -0,0 +1,8 @@
+argcomplete >= 3.2.1
+colorama
+colorlog
+openpyxl
+tabulate
+wcwidth
+xlrd
+xlwt
